course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -215,91 +436,131 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
+    "#### 1\\. Load data\n",
+    "Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# user code ..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3a"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "2a\\. Plot the following data in a single figure with three maps in one row:\n",
+    "----\n",
+    "\n",
+    "#### 2a\\. Plot E1, A1B and difference data\n",
```

Comment:
This question has quite a long answer! Perhaps we could split this into smaller questions:
1. Extract the year 2099 from A1B and E1.
2. Calculate the difference between the two.

Note: the question above just requires a loading of two cubes, so it's a bit unbalanced to go from small question to large question.